release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -997,6 +997,30 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 [id="ocp-4-9-cloud-compute-bug-fixes"]
 ==== Cloud Compute
 
+* Previously, when a root volume was created for a new server, and that server was not successfully created, the automatic deletion for the volume was not triggered because there was no deletion of a server associated with the volume. In some situations, this led to the creation of many extra volumes, and caused errors if the volume quota was reached. With this release, newly created root volumes are deleted when server creation call fails.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1943378[*BZ#1943378*])
+
+* Previously, when using the default value for `instanceType`, the Machine API created `m4.large` instances on AWS. This is different than the `m5.large` instance type for machines that are created by the {product-title} installer. With this release, the Machine API  creates `m5.large` instances for new machines on AWS when the default value is specified.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1953063[*BZ#1953063*])
+
+* Previously, the Machine API Operator constantly attached new targets even if they were already attached. The excessive calls to the AWS API resulted in a high number of errors. With this release, the Operator checks whether a load balancer attachment is required before attempting the attachment process. This change reduces the frequency of failed API requests.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1965080[*BZ#1965080*])
+
+* Previously, when using automatic pinning for a VM, the name of the property was `disabled`, `existing`, or `adjust`. With this release, the name better describes each policy, and `existing` was removed because it is blocked on oVirt. The new property names are `none` and `resize_and_pin`, which align with the oVirt user interface.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1972747[*BZ#1972747*])
+
+* Previously, the cluster autoscaler was unable to access the `csidrivers.storage.k8s.io` or `csistoragecapacities.storage.k8s.io` resources, which resulted in permissions errors. This fix updates the role assigned to the cluster autoscaler so that it includes permissions for these resources.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1973567[*BZ#1973567*])
+
+* Previously, it was possible to delete a machine with a node that has been deleted. This caused the machine to remain in a deleting phase indefinitely. This fix allows you to delete machines in this state properly.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1977369[*BZ#1977369*])
+
+* When using boot-from-volume image, creating a new instance leaks volumes if the machine-controller is rebooted. This caused the previously created volume to never be cleaned up. This fix ensures that the volume created previously is either pruned or reused.
```

Comment:
* boot-from-volume image -> the `boot-from-volume` image
* I think "machine-controller" should be "machine controller"